templates/emails/group.monthlyreport.hbs

```diff
@@ -28,43 +34,93 @@ ul li {
 .delta {
   color: #7A7B82;
 }
+.new {
+  color: #66C71A;
+}
+.lost {
+  color: red;
+}
 .label {
 
 }
 .reimbursed {
   color: #7A7B82;
 }
+.tdline {
+  border-bottom: 1px solid #DDE1E4; /* #7A7B82; */
+  width: 40%;
+  height: 20px;
+}
+
+th {
+  padding: 10px;
+  font-weight: normal;
+  font-size: 14px;
+  color: #7A7B82;
+  vertical-align: top;
+}
 </style>
 
 <div style="font-size:10pt">
 
 <h1>{{group.name}} {{month}} report</h1>
 
-<p>Hi there! This is the monthly report for your open collective with your latest balance (as of the last day of last month) and all the expenses that were submitted.
-You can always visit your collective page to get the latest up to date data: <a href="{{group.publicUrl}}">{{group.publicUrl}}</a><br />
-Don't hesitate to send us your feedback by replying to this email, we thrive on feedback! ❤️ ❤️</p>
+<p>Hi!<br />
+We have so much great news this month, we could barely wait to share them with you! Please remember, don't hesitate to send us your feedback by replying to this email, we thrive on feedback! ❤️ ❤️</p>
```

Comment:
Second sentence "Please don't hesitate to send us feedback - just reply to this email! ❤️ ❤️</p>"